rita/src/exit.rs

```diff
@@ -161,6 +162,7 @@ fn main() {
         App::new()
             .resource("/setup", |r| r.method(Method::POST).with2(setup_request))
             .resource("/list", |r| r.method(Method::POST).with(list_clients))
+            .resource("/exit_info", |r| r.method(Method::GET).with(get_exit_info))
```

Comment:
What's the point of a separate endpoint for this? Couldn't this just be `/setup` with GET?

Reply:
getting setup doesn't really make sense to me, as it is general info about the exit

Reply:
Well, it's setup both ways - for the client when you `GET` and for the exit when you `POST`